db/migrate/20190130090437_add_crop_to_photographings.rb

```diff
@@ -0,0 +1,10 @@
+class AddCropToPhotographings < ActiveRecord::Migration[5.2]
+  def change
+    add_column(:photographings, :crop_id, :integer)
+    add_foreign_key(:photographings, :crops)
+
+    Photographing.where(crop_id: nil).each do |p|
```

Comment:
I've checked and we have 2500 records in this table on line, which should be fine for this line of code to cope with